cmd/cli/plugin/package/main.go

```diff
@@ -4,6 +4,7 @@
 package main
 
 import (
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
The  imports should be grouped  in three sets: std lib and go0internal imports, external imports, local vmware-tanzu imports, and each set should be separated by an empty line. (In this case, ```"k8s.io/client-go/tools/clientcmd"``` can be moved to where ```"github.com/aunum/log"``` is, followed by a ```make fmt``` to fix the formatting

Reply:
thanks. done